pages/_app.jsx

```diff
@@ -27,14 +27,14 @@ import { MatomoProvider, createInstance } from "@datapunt/matomo-tracker-react";
 
 import PropTypes from "prop-types";
 import Head from "next/head";
-import { useRouter } from "next/router";
+import router, { useRouter } from "next/router";
```

Comment:
Heh, this is interesting. This seems to work, but I can't find anything in [the docs](https://nextjs.org/docs/api-reference/next/router) about a default export from `next/router`. I think to be sure it's probably safer to align with the docs and call `.push` on the return value of `useRouter`.

Reply:
Oh that's so weird. Probably auto-imported and I didn't notice.